e2e/app/page-mixin/PageNavigation.ts

```diff
@@ -0,0 +1,95 @@
+import {Page} from 'puppeteer';
+import {iconXpath} from '../aou-elements/xpath-defaults';
+import {findIcon} from '../aou-elements/xpath-finder';
+
+
+export const LINK = {
```

Comment:
Are all of these actively used? Or is this just trying to be exhaustive?

If you haven't had it already, I think we should have a discussion about how we should think about keeping these things in sync with the application, especially if the goal is to have 100% UI coverage of the different elements.

For example, if we're depending on text labels like this, it might be more maintainable to export specific values/labels from our React components. This will result in a quick compile-time issue, rather than a runtime integration test failure, which might take a very long time to manifest.